utils/optiwebp.py

```diff
@@ -0,0 +1,160 @@
+                        continue
+            elif filetype == ".jpg":
+                subprocess_args = lossy_args
+            else:
+                continue
+
+            initial_file = os.path.join(image_root, filename)
+            webp_file = ""
+            if filetype != ".webp":
+                webp_file = os.path.join(options.tempdir, filename)[:-3]+"webp"
+            else:
+                webp_file = os.path.join(options.tempdir, filename)
+
+            result = subprocess.run(subprocess_args + [initial_file, "-o", webp_file], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+            if result.returncode != 0:
+                print(initial_file+" failed conversion to webp.")
+                continue
+
+            initial_size = os.path.getsize(initial_file)
+            converted_size = os.path.getsize(webp_file)
+            percentage_change = round((converted_size/initial_size)*100, 2)
+            absolute_change = initial_size - converted_size
+
+            results_dict[initial_file] = {
+                "type": filetype,
+                "new_file": webp_file,
+                "old_size": initial_size,
+                "new_size": converted_size,
+                "change_in_bytes": absolute_change,
+                "change_in_percent": percentage_change
+            }
+
+            if options.nodryrun and percentage_change < options.minpercent:
+                os.remove(initial_file)
+                shutil.copy(webp_file, image_root)
+
+            os.remove(webp_file)
+
+good_conversion = "good-conversion.tsv"
+bad_conversion = "bad-conversion.tsv"
+
+if os.path.exists(good_conversion):
+    os.remove(good_conversion)
+with open(good_conversion, "a") as f:
+    f.write("filename,old_size,new_size,change_in_percent,change_in_bytes\n")
+
+if os.path.exists(bad_conversion):
+    os.remove(bad_conversion)
+with open(bad_conversion, "a") as f:
+    f.write("filename,old_size,new_size,change_in_percent,change_in_bytes\n")
+
+initial_total_size = 0
+final_total_size = 0
+
+with open(good_conversion, "a") as good_f, open(bad_conversion, "a") as bad_f:
+    for filename, data in results_dict.items():
+        line = "{filename}\t{old_size}\t{new_size}\t{change_in_percent}\t{change_in_bytes}\n".format(filename=filename, old_size=data["old_size"], new_size=data["new_size"], change_in_percent=data["change_in_percent"], change_in_bytes=data["change_in_bytes"])
+
+        if data["change_in_percent"] < options.minpercent:
+            initial_total_size = initial_total_size + data["old_size"]
+            final_total_size = final_total_size + data["new_size"]
+
+            good_f.write(line)
+        else:
+            bad_f.write(line)
+
+total_percentage_change = round((final_total_size/initial_total_size)*100, 2)
+duration = time.time() - start
+
+hours_duration = int(duration / 3600)
+minutes_duration = int((duration - (hours_duration * 3600)) / 60)
```

Comment:
Again, using the modulo operator is better: `minutes_duration = int((duration % 3600) / 60)`.
If you want, you can also remove the `int()` casting and use the integer division operator `//` (example: `hours_duration = duration // 3600`

Reply:
Done.